subworkflows/local/generate_reports/main.nf

```diff
@@ -129,12 +128,12 @@ workflow GENERATE_REPORTS {
         ch_report_inputs.graph,
         ch_report_inputs.annotate,
         ch_report_inputs.analysis,
-        ch_report_inputs.layout
+        ch_report_inputs.layout,
```

Comment:
Does the trailing comma's here work? I thought it wasn't allowed.

Reply:
I think so. But I will remove it just to be sure.